tests/unit_tests/env_vars.test.js

```diff
@@ -0,0 +1,53 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const env_vars = require("../../lib/utils/env_vars.js");
+const validateEnvironment = env_vars.validateEnvironment;
```

Comment:
TODO: Make an issue to mock the `process.env` vars.

Reply:
Or just set the values.

Reply:
So that tests don't depend on previous values.

Reply:
Here is new [Issue 377](https://github.com/plocket/docassemble-cucumber/issues/377) to cover this area of concern.